Swarm/src/config/envs/local.ts

```diff
@@ -24,10 +25,6 @@ export function createLocalConfig(): SystemConfig {
       configuration: {
         heartbeatInterval: 60,
         bootRetryInterval: 15,
-        unknownChargerStatus: RegistrationStatusEnumType.Accepted,
-        getBaseReportOnPending: true,
-        bootWithRejectedVariables: true,
-        autoAccept: true,
```

Comment:
Could you pls reset these config? It is still needed before merging the new boot changes.